documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -156,43 +148,46 @@ for a comprehensive list of options.
 [[running-tests-build-gradle-bom]]
 ===== Aligning dependency versions
 
+TIP: See <<running-tests-build-spring-boot>> for details on how to override the version
+of JUnit used in your Spring Boot application.
+
 Unless you're using Spring Boot which defines its own way of managing dependencies, it is
 recommended to use the JUnit Platform BOM to align the versions of all JUnit 5 artifacts.
 
 [source,groovy,indent=0]
 [subs=attributes+]
+.Explicit platform dependency on the BOM
 ----
 dependencies {
 	testImplementation(platform("org.junit:junit-bom:{bom-version}"))
+    testImplementation("org.junit.jupiter:junit-jupiter")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
 }
 ----
 
 Using the BOM allows you to omit the version when declaring dependencies on all artifacts
 with the `org.junit.platform`, `org.junit.jupiter`, and `org.junit.vintage` group IDs.
 
-TIP: See <<running-tests-build-spring-boot>> for details on how to override the version
-of JUnit used in your Spring Boot application.
-
-[[running-tests-build-gradle-config-params]]
-===== Configuration Parameters
-
-The standard Gradle `test` task currently does not provide a dedicated DSL to set JUnit
-Platform <<running-tests-config-params, configuration parameters>> to influence test
-discovery and execution. However, you can provide configuration parameters within the
-build script via system properties (as shown below) or via the
-`junit-platform.properties` file.
+Since all JUnit artifacts declare a
+https://docs.gradle.org/current/userguide/platforms.html[platform] dependency on the BOM,
+you usually don't need to declare an explicit dependency on it yourself. Instead, it's
+sufficient to declare one dependency including a version to be able to omit the version
```

Comment:
I know what you mean here, but I fear it might not be clear to the average reader.

Perhaps rephrasing "one dependency including a version" would help.

Or, maybe it's better to mention when dependency in the following example achieves that goal.

Reply:
I've expanded this part and update the example below in e709a0ae8e2a535a8827aaa66f505825b6c9915f.